net/instaweb/rewriter/public/rewrite_options.h

```diff
@@ -1527,6 +1528,13 @@ class RewriteOptions {
     set_option(x, &blink_blacklist_end_timestamp_ms_);
   }
 
+  bool preserve_subresource_hints() const {
```

Comment:
I think the phrasing is right, because stripping (if needed) is a default behavior and we also have some preserve options already. 
@jeffkaufman @oschaaf @jmarantz , any thoughts?